### PR TITLE
WIP Storybook Improvements

### PR DESCRIPTION
**Pull request by bradfordjanice**
Continued Storybook improvements:

Organizing stories
Adding Style information

### CODE REVIEW

**Comment by jaronheard** [comment]
@bradfordjanice @sixfoot6 
Here are some links to some supporting documents. Both of these could use some editing.
[What is a card?](https://docs.google.com/document/d/1Zbqx92ARaJGkwob6K8DG_faifbfIwFO4qx7h8LcCf6s/edit#) - 
[Data visualization guidelines](https://github.com/hackoregon/civic/blob/master/DATAVIZ.MD)

**Comment by bradfordjanice** [comment]
Added the Logo and Typography sections to the Style Guide in Storybook.
Two svg logo files need to be completed.
Some of the global CSS styles need to be applied; these are noted in the content.

File: packages/component-library/stories/Introduction.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      or more UI components. (Basically a story is like a visual test case.)
+    </p>
+    <p>See these sample stories for a component called Button.</p>
+    <p>
+      You can add your own components as stories. You can also edit those
+      components and see changes right away. Try editing the Button component.
+    </p>
+    <p>
+      Our Storybook is iterative and will be updated as the CIVIC Platform
+      continues to grow. Please reach out to us in the SLACK #product_design
```
Comment (jaronheard) [inline packages/component-library/stories/Introduction.js]: change SLACK to #team-design

File: packages/component-library/stories/TheBrandStyle.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      <li>Data for the People by the People</li>
+      <li>Story over Framework</li>
+      <li>Usable over Pretty</li>
+      <li>Neutrality over Partisanship</li>
+      <li>Inclusion over Specificity</li>
+      <li>Clarity over Saturation</li>
+    </ul>
+    <p>
+      Our Storybook is iterative and will be updated as the CIVIC Platform
+      continues to grow. Please reach out to us in the SLACK #product_design
```
Comment (jaronheard) [inline packages/component-library/stories/TheBrandStyle.js]: change SLACK to #team-design

File: packages/component-library/stories/Introduction.js (hunk 1 above)
Comment (bradfordjanice) [reply packages/component-library/stories/Introduction.js]: Done.

File: packages/component-library/stories/TheBrandStyle.js (hunk 2 above)
Comment (bradfordjanice) [reply packages/component-library/stories/TheBrandStyle.js]: Done.

**Comment by bradfordjanice** [comment]
Added Storybook stories for Logo and Typography styles.

**Pull request merged**